src/auth/socialAuth/google/css/Google.scss

```diff
@@ -0,0 +1,16 @@
+.googlebutton > button {
+    box-shadow: 0px 0px 0px 0px !important;
```

Comment:
@neelxie wherever there is 0 as a value, remove a metric. i.e set `margin:0` instead of `margin:0px`

Reply:
@alexxsanya this is a mandatory requirement for the google button, and as you can tell, it is only available for the google button.